etc/configuration/default.json

```diff
@@ -260,7 +260,7 @@
     "initial_covariance": [0.25, 0.25],
     "measurement_count_threshold": 10,
     "use_feet_detection_measurements": true,
-    "use_robot_detection_measurements": true,
+    "use_robot_detection_measurements": false,
```

Comment:
Please also add the new parameters in the `default.json` file.

Reply:
Done!

Reply:
Done!